storybook/pages/ModelEntryPage.qml

```diff
@@ -20,6 +22,32 @@ Control {
         sourceModel: usersModel
         key: "pubKey"
         value: pubKeySelector.currentText
+        
+        onItemChanged: signalsModel.append({        signal: "Item changed",     value: "", row: itemData.row, item: itemData.item, roles: JSON.stringify(itemData.roles), available: itemData.available})
```

Comment:
[Screencast from 19.07.2024 11:49:46.webm](https://github.com/user-attachments/assets/572aa180-ddb3-4445-ac01-f896c6d619ed)

One small doubt here. When item removed, shouldn't be `available` set to false first?

There is also warning printed, but it comes from `GenericListView`, can be fixed by adding `Component.onDestruction: loader.active = false` in delegate.